src/pokemon-service/main.go

```diff
@@ -8,15 +8,19 @@ import (
 	"github.com/GerardoHP/ondemand-go-bootcamp/registry"
 	"github.com/GerardoHP/ondemand-go-bootcamp/router"
 	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
 )
 
 // Starts the echo server with the available methods
 func main() {
-	config := config.GetInstance()
+	config := config.NewConfig()
```

Comment:
Normally the New funcs, are named just New, to avoid redundancy:
`config.NewConfig` ---> `config.New`

Reply:
Attended wherever it applies